org.civicrm.reqangular/src/common/services/api.js

```diff
@@ -9,6 +9,20 @@ define([
 
     var API_ENDPOINT = '/civicrm/ajax/rest';
 
+    /**
+     * Common handler for API calls
+     *
+     * @param {object} response - response from backend
+     * @return {Promise/Any}
+     */
+    function commonHandler(response) {
```

Comment:
`commonHandler` was just a placeholder name that i was using in my example....it should go without saying that you should give it a descriptive name